cmd/deploy.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/unweave/cli/config"
+	"github.com/unweave/cli/ui"
+)
+
+func Deploy(cmd *cobra.Command, args []string) error {
+	return runSSHConnectionCommand(cmd, args, &deployCommandFlow{})
+}
+
+type deployCommandFlow struct {
+	execCommandFlow
+}
+
+func (d *deployCommandFlow) parseArgs(cmd *cobra.Command, args []string) execCmdArgs {
+	if len(args) == 0 {
+		const errMsg = "❌ Invalid arguments. You must pass a file path to deploy.\n" +
+			"See `unweave deploy --help` for more information"
+		ui.Errorf(errMsg)
+		os.Exit(1)
```

Comment:
I'm guessing this is temporary? We shouldn't need this restriction once ready

Reply:
It's based on (my interpretation of) the spec, I can easily change it.

Would `deploy` without a filepath skip the copy?

Reply:
Yeah I don't see a reason where the user must specify a file. Maybe the file is already present on the container image. If not, the build will fail and that's fine as long as we tell the user why.